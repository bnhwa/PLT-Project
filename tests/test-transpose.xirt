num main() {
    xirtam m;
    m = [[1, 2], [3, 4]];

    printm(trans(m));
}