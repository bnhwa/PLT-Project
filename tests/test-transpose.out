1.00 3.00
2.00 4.00